### PR TITLE
Added clean up section and fix keyfile permession warning

### PR DESCRIPTION
**Pull request by martinliu**
This article should have a "clean up" section. We need to tell users how to clean up the 12 objects tf have created.
Also, we need to put the public key to a path to set 600 permission for it. Otherwise, ssh connect will be failed due to a permission warning below.
```
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions 0777 for 'id_rsa' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
Load key "id_rsa": bad permissions
```

### CODE REVIEW

File: articles/terraform/create-linux-virtual-machine-with-infrastructure.md [hunk 1]
```diff
@@ -248,7 +248,7 @@ To use SSH to connect to the virtual machine, do the following steps:
 1. Run [terraform output](https://www.terraform.io/cli/commands/output) to get the SSH private key and save it to a file.
 
     ```console
-    terraform output -raw tls_private_key > id_rsa
+    terraform output -raw tls_private_key > ~/id_rsa
```
Comment (martinliu) [inline articles/terraform/create-linux-virtual-machine-with-infrastructure.md:251]: If we put id_rsa inside cloudsorage path, we can not change the permission to 600. But, ~/ works.

File: articles/terraform/create-linux-virtual-machine-with-infrastructure.md [hunk 2]
```diff
@@ -261,9 +261,15 @@ To use SSH to connect to the virtual machine, do the following steps:
 1. Use SSH to connect to the virtual machine.
 
     ```console
-    ssh -i id_rsa azureuser@<public_ip_address>
+    chmod 600 ~/id_rsa
```
Comment (martinliu) [inline articles/terraform/create-linux-virtual-machine-with-infrastructure.md:264]: this private key need 600 to work with ssh connection.

File: articles/terraform/create-linux-virtual-machine-with-infrastructure.md [hunk 3]
```diff
@@ -261,9 +261,15 @@ To use SSH to connect to the virtual machine, do the following steps:
...
+    ssh -i ~/id_rsa azureuser@<public_ip_address>
     ```
 
+## Clean up resources
```
Comment (martinliu) [inline articles/terraform/create-linux-virtual-machine-with-infrastructure.md:268]: After I walked this article to the end, I really miss this clean-up session to destroy everything tf have created before.

**Comment by TomArcherMsft** [comment]
@martinliu This article has been rewritten and should publish next week.

**Pull request closed without merging**